gradle.properties

```diff
@@ -17,4 +17,10 @@ org.gradle.parallel=true
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:configuration_on_demand
 org.gradle.configureondemand=false
 
-android.enableUnitTestBinaryResources=true
\ No newline at end of file
+android.enableUnitTestBinaryResources=true
+
+# use the appropriate AndroidX library instead of a Support Library.
+android.useAndroidX=true
+
+# automatically migrates existing third-party libraries to use AndroidX by rewriting their binaries
```

Comment:
is this impact only around android support libraries or does it go further than that?

Reply:
Android X replaces only Support libs and it's API's. So the impact is limited to that.